protos/cerbos/cloud/store/v1/store.proto

```diff
@@ -167,22 +156,17 @@ message ReplaceFilesRequest {
   optional ChangeDetails change_details = 4;
 }
 
-message ReplaceFilesResponse {
-  message Failure {
-    repeated FileError errors = 1;
-    string cause = 2;
-  }
+message ErrDetailValidationFailure {
+  repeated FileError errors = 1;
+}
 
-  message Success {
-    int64 new_store_version = 1;
-  }
+message ErrDetailUnusableFiles {
```

Comment:
I slightly prefer the name of the error in the implementation code because it covers the case of an empty zip file, too, right?

```suggestion
message ErrDetailNoUsableFiles {
```